### PR TITLE
Fixed NAT gateway setup for EKS examples

### PR DESCRIPTION
**Pull request by andrejatcastai**

### CODE REVIEW

File: examples/eks_cluster_assumerole/main.tf [hunk 1]
```diff
@@ -12,6 +12,7 @@ provider "helm" {
 
 provider "castai" {
   api_token = var.castai_api_token
+  api_url   = var.castai_api_url
```
Comment (aldor007) [inline examples/eks_cluster_assumerole/main.tf:15]: `castai_api_url` is not defined in variables.tf

Comment (andrejatcastai) [reply examples/eks_cluster_assumerole/main.tf:15]: Damn, did not push it along with me local changes, fixed. Thanks ;)

File: examples/eks_cluster_assumerole/eks.tf [hunk 2]
```diff
@@ -4,10 +4,10 @@ module "eks" {
   version = "17.24.0"
 
   cluster_name    = var.cluster_name
-  cluster_version = "1.21"
+  cluster_version = "1.22"
```
Comment (aldor007) [inline examples/eks_cluster_assumerole/eks.tf:7]: Why not 1.23? :)

**Review by aldor007 — COMMENTED** [review]
Configuration for aws provider is not required? 
```
provider "aws" {
  region = var.cluster_region
}
```

File: examples/eks_cluster_assumerole/eks.tf (hunk 2 above)
Comment (andrejatcastai) [reply examples/eks_cluster_assumerole/eks.tf]: Changed and tested with 1.23 ;)

**Comment by andrejatcastai** [comment]
Tested once again and it works without
```
provider "aws" {
  region = var.cluster_region
}
```

**Review by aldor007 — APPROVED** [review]

**Pull request merged**